frontend/src/pages/Lookup.jsx

```diff
@@ -1,218 +1,175 @@
-              {workflows.map((workflow) => {
-                return <option className="text-center">{workflow.name}</option>
-              })}
-              </select>
-              ) : (
-                <div></div>
+        <div className="container">
+          <div className="form-container">
+            <form onSubmit={getWorkflowData} className="form" id="myForm">
+                <div className="type-ahead-container">
+                    <label for="repositoryName">Repository Name:</label>
+                    <input value={selected} type="text" id="repositoryName" name="repository" className="type-ahead-input" placeholder="Type to search..." required
+                      onChange={filterRepositories}
+                      onFocus={(event) => {
+                        event.target.setAttribute('autocomplete', 'off');
+                      }}
+                    />
+                    <ul className="type-ahead-dropdown">
+                      {suggestedRepositories.map((repo) => {
+                        return <li className="type-ahead-item"
+                                key={repo.id} onClick={() => {
+                                  setSelectedFromList(repo);
+                                }}>{repo.name}</li>;
+                      })}
+                    </ul>
+                </div>
+
+                <div className="select-workflow">
+                    <label>Select Workflow:</label>
+                      {
+                      workflows
+                        ? (
+                        <select id="selectOption" name="workflow" required>
+                          {Object.keys(workflows).map((workflowName) => {
+                            return <option className="text-center">{workflowName}</option>;
+                          })}
+                        </select>
+                          )
+                        : (
+                      <div></div>
+                          )
+                    }
+                </div>
+                <button type="submit">Search</button>
+            </form>
+          </div>
+          {
+          (jobBreakDown.length) > 0
+            ? (
+            <div className="placeholder-container">
+              <div className="placeholder">
+                <h2> Most Frequently Broken </h2>
+                <Table
+                  TableRows={Object.keys(failureCountMap).map((k) => (
+                    <TableRow rowData={[k, failureCountMap[k]]}></TableRow>
+                  ))
+                  }
+                  Header=<TableHeader headers={['job name', 'times broken']}/>
+                  tableClassName="styled-table"
+                />
+              </div>
+              <div className="placeholder">
+                <h2>Graph Breakdown</h2>
+                  <Graph repository={repository} jobBreakDown={jobBreakDown} workflowName={workflow} />
+              </div>
+            </div>
+              )
+            : (
+            <div></div>
```

Comment:
do you want an empty div or do you want nothing at all rendered? you can do 

`{(jobBreakdown.length > 0 && <div>my content</div>}` and react will just render nothing if length is 0 or less